src/app/feature/map/map.component.html

```diff
@@ -13,11 +13,7 @@
     <app-floor-switcher class="floor-switcher"></app-floor-switcher>
   </div>
   <div class="right-section">
-    <button class="manual" (click)="openDialog()" matRipple>
-      <svg class="icon">
-        <use xlink:href="assets/icons/sprite.svg#info"></use>
-      </svg>
-    </button>
+    <app-manual-button class="manual"></app-manual-button>
```

Comment:
Please check, we need to decide how we rename `app` prefix.
Example: `gmp`, `guidemap`